lib/web_ui/test/keyboard_test.dart

```diff
@@ -453,6 +453,7 @@ void main() {
             key: 'i',
             code: 'KeyI',
             repeat: true,
+            isMetaPressed: true,
```

Comment:
This test is trying to simulate the case where the meta keys have been released. I think we shouldn't use `isMetaPressed: true` because the meta key has been released already (see line 436 above).